### ISSUE TITLE
reddit.com some new advertisements not blocked

### ISSUE THREAD

**Issue by frank1iu** [uBlockOrigin/uAssets#677]
### URL(s) where the issue occurs

`https://reddit.com/`

### Describe the issue

New advertisements called "recommendations" not blocked.

### Screenshot(s)

https://i.imgur.com/10YpS3G.png

### Versions

- Browser/version: Mozilla Firefox 55.0.3
- uBlock Origin version: 1.14.4

### Notes

Seems to be blockable with `reddit.com##.recommended-link`

Edit: Reddit's changelog on this new type of ad: https://redd.it/5zrtnr

**Comment by toshiya44** [comment]
I thought this was already fixed? 

Anyway, these are the scripts that usually load promoted content. 
```
||reddit.com/api/request_promo$important
||reddit.com/api/request_recommended_link$important
||redditstatic.com/recommended-link.*.js$important
```

**Comment by gotitbro** [comment]
@toshiya44 No need for the `$important` modifier in these filters. The first filter is already there in EasyList, that URL is used to fetch ads on the mobile website (More info: https://github.com/uBlockOrigin/uAssets/issues/475).

The ads reportedly been seeing by @frank1iu are the "recommended" ones which appear inconsistently and already have an open issue https://github.com/uBlockOrigin/uAssets/issues/493. They are blocked by the second filter above no need for the third one. I am not sure why this has not been fixed till now.

**Comment by frank1iu** [comment]
Closed for being a duplicate of #493.
